app/admin/widgets/menu/assets/js/mainmenu.js

```diff
@@ -80,6 +80,9 @@
         if (!$itemMenu.length)
             return;
 
+        if (window.matchMedia('(max-width: 600px)'))
+            $('.sidebar, .nav-sidebar').collapse('hide')
+
```

Comment:
Needs to be before the `!$itemMenu.length` conditional to apply on all menu items

Reply:
Ok, updated